src/backend/renderer.js

```diff
@@ -743,23 +743,26 @@ export function attach(
     }
   }
 
-  function mountFiber(fiber: Fiber, parentFiber: Fiber | null) {
+  function mountFiber(
+    fiber: Fiber,
+    parentFiber: Fiber | null,
+    traverseSiblings = false
```

Comment:
I originally wanted to split into two functions but that felt a bit more confusing. I'm ok with this signature but I can split if you want.